src/sycl/CMakeLists.txt

```diff
@@ -74,10 +74,12 @@ if(WITH_ROCM_BACKEND)
 endif()
 
 if(WITH_CPU_BACKEND)
+  find_package(OpenMP REQUIRED)
   add_library(hipSYCL_cpu SHARED ${HIPSYCL_SOURCES})
   target_compile_definitions(hipSYCL_cpu PRIVATE HIPSYCL_BOOTSTRAP=1)
   target_compile_options(hipSYCL_cpu PRIVATE --hipsycl-platform=cpu ${COMMON_SYCLCC_OPTIONS})
-  target_include_directories(hipSYCL_cpu PRIVATE ${INCLUDE_DIRS} ${HIPSYCL_SOURCE_DIR}/contrib/hipCPU/include)
+  target_include_directories(hipSYCL_cpu PRIVATE ${INCLUDE_DIRS} ${OpenMP_CXX_INCLUDE_DIRS} ${HIPSYCL_SOURCE_DIR}/contrib/hipCPU/include)
```

Comment:
Again, I wonder if we can make this specific only to Apple, so that the other platforms can continue to support older cmake versions.

Reply:
... and can you elaborate why we need to add the OpenMP include directories when building hipSYCL, but not when later compiling programs using `syclcc`? There it seems to be sufficient to add `-Xclang`?

Reply:
will just check here for cmake 3.16+ for now.

we need them in both cases on macOS, in which line did I miss to add the includes?

Reply:
Nevermind, I get it now - `syclcc` doesn't add the includes, these get added by cmake, and when compiling with `syclcc` later on, the user must manually specify the OpenMP include paths.